google-cloud-spanner/src/test/java/com/google/cloud/spanner/OptionsTest.java

```diff
@@ -170,10 +171,13 @@ public void listEquality() {
   @Test
   public void readOptTest() {
     int limit = 3;
-    Options opts = Options.fromReadOptions(Options.limit(limit));
+    String tag = "app=spanner,env=test,action=read";
+    Options opts = Options.fromReadOptions(Options.limit(limit), Options.tag(tag));
```

Comment:
nit: please don't abbreviate `Options` to `opts`. (I know this was already like this, but if you are to make any other changes, could you please fix this as well).

Reply:
Done.